app/server/app/public/widgets/community-widgets.html

```diff
@@ -79,8 +79,7 @@ <h3>How’s My Waterway Widget (220px &times; 170px)</h3>
   title="How’s My Waterway Widget"
   width="220"
   height="170"
-  frameborder="0"
-  scrolling="no"&gt;
+  style="border: none; overflow: hidden;"&gt;
 &lt;/iframe&gt;
```

Comment:
I think I left the depreciated `frameborder` and `scrolling` attributes in the iframe because that's what YouTube embeds do. I know I looked into it at the time (which was only a few weeks ago, but I've forgotten what the exact reason was for me to keep it like this was).

What you've done is technically correct, but since we already delivered this code to EPA (and Mike posted it on EPA's website) I think we should probably keep it as is, with the deprecated attributes in there.

Reply:
Sounds good. I'll go ahead and revert that code.